src/app/content/highlights/components/Note.tsx

```diff
@@ -36,7 +36,7 @@ const TextArea = styled.textarea`
 
 // tslint:disable-next-line:variable-name
 const Note = ({onChange, onFocus, note, textareaRef}: Props) => {
-  const setTextAreaHeight = () => {
+  const setTextAreaHeight = React.useCallback(() => {
```

Comment:
Why did you make this change?

Reply:
`react-hooks/exhaustive-deps` error in useEffect below